content.yml

```diff
@@ -55,12 +55,13 @@ easy_access:
   visible: true
   title: "For crypto nerds, finance nerds and non-nerds at all"
   paragraph:  |
-        Easy to use crafted experience for different actors on different stages of their investment journey. An on-going work, being build with community for community,
-        open-source code, access point for everyone to blockchain.
+        Easy to use crafted experience for different actors on different stages of their investment journey. An
+        on&#8209;going work, being built with community for community, open&#8209;source code, access point for everyone
```

Comment:
what about `&#8209;`? how using `-` behaves?

Reply:
It is non breaking hyphen. Without it there is break in on-going text.

Reply:
uhh nice :D til 👍